src/main/java/com/dev/cinema/config/AppConfig.java

```diff
@@ -14,7 +14,9 @@
 @PropertySource("classpath:db.properties")
 @ComponentScan(basePackages = {
         "com.dev.cinema.service",
+        "com.dev.cinema.security",
         "com.dev.cinema.dao"
+
```

Comment:
don't add this empty line, it's redundant

Reply:
done